activejob/lib/active_job/logging.rb

```diff
@@ -4,17 +4,24 @@
 require "active_support/logger"
 
 module ActiveJob
-  module Logging # :nodoc:
+  module Logging
     extend ActiveSupport::Concern
 
     included do
+      ##
+      # :singleton_method:
+      #
+      # Accepts a logger conforming to the interface of Log4r or the default
+      # Ruby +Logger+ class. You can retrieve this logger by calling +logger+ on
+      # either an Active Job job class or an Active Job job instance.
```

Comment:
This will make `ActiveJob::Logging` visible in the API docs tree.
Maybe move the comment to `ActiveJob::Base` instead?

Reply:
Why not document `log_arguments` as well?

Reply:
I've added a doc for `log_arguments` now, no reason I hadn't before other than I was looking at `logger` in particular.

Re: `ActiveJob::Logging`, I don't have a strong opinion. It looks like this class is similar to `AbstractController::Logger` (which maybe we should document as well?) in that it adds a logger to a class that includes it, but a difference with the `AbstractController` module is that `ActionController::Metal` does not include it by default (and Active Job doesn't really have a `Metal` equivalent). 

Since `ActiveRecord::Core` defines it's logger method (not module being included), maybe this module should be inlined as well? I think that would make more sense to me than documenting 2/3 of the public methods in this module on another class